trivup/apps/SchemaRegistryApp.py

```diff
@@ -28,96 +28,115 @@
 
 from trivup import trivup
 from trivup.apps.KafkaBrokerApp import KafkaBrokerApp
+from trivup.apps.SslApp import SslApp
 
-import uuid
 import requests
-import subprocess
 
 
-class SchemaRegistryApp (trivup.App):
+sr_image = 'confluentinc/cp-schema-registry'
+
+
+class SchemaRegistryApp (trivup.DockerApp):
     """ Confluent Schema Registry app.
         Depends on KafkaBrokerApp.
         Requires docker. """
 
-    default_image = 'confluentinc/cp-schema-registry:5.2.1'
+    default_version = '5.3.0'
+
+    default_app_conf = {
+        'debug': True,
+        'host.name': 'localhost',
+        'listeners': 'http://0.0.0.0:8081'
+    }
 
-    def __init__(self, cluster, conf=None, on=None):
+    @staticmethod
+    def _conf_to_env(app_conf):
+        return ["SCHEMA_REGISTRY_{}={}"
+                .format(key.replace(".", "_").upper(), value)
+                for key, value in app_conf.items()]
+
+    def __init__(self, cluster, conf=None):
         """
         @param cluster     Current cluster
         @param conf        Configuration dict, see below.
         @param on          Node name to run on
 
         Supported conf keys:
            * version - Confluent Platform version to use.
+           * enable_auth - Enable Http basic auth
            * port_base - Low TCP port base to start allocating from (random)
-           * image - docker image to use
```

Comment:
why remove the ability to specify an alternate image?

Reply:
In practice we never pass an alternative image so this seemed simpler. Can revert though